pkgs/tools/virtualization/cloud-init/default.nix

```diff
@@ -116,6 +117,15 @@ python3.pkgs.buildPythonApplication rec {
 
   pythonImportsCheck = [
     "cloudinit"
+    "configobj"
+    "jinja2"
+    "jsonpatch"
+    "jsonschema"
+    "netifaces"
+    "oauthlib"
+    "yaml"
+    "requests"
+    "serial"
```

Comment:
Most module are already executing `pythonImportsCheck` on their own (e.g., https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/netifaces/default.nix#L17).

```suggestion
```

Reply:
`pythonImportsCheck` is only an additional safe-guard. If there are unit tests available then it would already be covered. However `pythonImportsCheck` is cheap and primarily for modules but it works for Python applications too.

For Python application sometimes checking if the tool run can be helpful to catch missing/new/changed dependencies. E.g., something like: 

```nix
  installCheckPhase = ''
    runHook preInstallCheck
    $out/bin/cloud-init --version | grep "v${version}"
    runHook postInstallCheck
  '';
```